.gitignore

```diff
@@ -0,0 +1,7 @@
+.settings/
+target/
+.project
+.classpath
+**/.flattened-pom.xml
+/bin/
+.README.md.html
```

Comment:
where does this html even come from? You can delete it from `.gitignore` and from the project